### ISSUE TITLE
Add a dist command to the npm package.json file in order to use the lib with a browser.

### ISSUE THREAD

**Issue by Natim** [Kinto/kinto-discovery-lib#4]
@n1k0 could you tell us how we should do that please?

**Comment by n1k0** [comment]
```
16:50:21 <NiKo`> natim, schweta: how is the discovery lib planned to be used in browser?
16:50:23 <NiKo`> I realize there's possibly a little more than packaging from what I'm seeing
16:51:10 <NiKo`> let me put that another way: what's the expected public API exposed to the browser when the lib is loaded?
16:51:32 <NiKo`> what would be great is to have a little Usage section in the readme
16:51:56 <NiKo`> so I know which symbols I'm supposed to expose to the browser
```

**Comment by n1k0** [comment]
See #7.

**Comment by Natim** [comment]
Fixed in #7
